Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Reduce unemployment with NFT Skills Certification
+author: Diki karim @dikikarim2010, Diki karim dikikarim2010@gmail.com
+company_name: none
+category: dApps
+focus_area: Education,NFT,Social
+open_source: Yes
+amount_requested: 1000000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.

Reply:
thank you for your attention